src/python/lib/clusterfuzz/stacktraces/__init__.py

```diff
@@ -674,9 +674,10 @@ def parse(self, stacktrace: str) -> CrashInfo:
         continue
 
       # Sanitizer regular crash (includes ills, abrt, etc).
-      # However if we have a Go, Python, or Kernel crash, don't replace
-      # this with sanitizer signal handler failure.
+      # Certain crash types have precedence though; don't overwrite them.
       if (not state.crash_type.startswith('Kernel failure') and
+          not state.crash_type in
+            IGNORE_CRASH_TYPES_FOR_ABRT_BREAKPOINT_AND_ILLS and
```

Comment:
Can you modify line 422 onwards instead? Just need to add a new pattern for TRAPs.

```python
      # Ignore aborts, breakpoints and ills for asserts, check and dcheck
      # failures. These are intended, retain their original state.
      if (SAN_ABRT_REGEX.match(line) or SAN_BREAKPOINT_REGEX.match(line) or
          SAN_ILL_REGEX.match(line)):
        if state.crash_type in IGNORE_CRASH_TYPES_FOR_ABRT_BREAKPOINT_AND_ILLS:
          continue
```

Reply:
Ah, didn't find that location. It indeed fits nicer there. Thanks!